src/sentry/utils/committers.py

```diff
@@ -130,6 +130,34 @@ def _get_committers(annotated_frames, commits):
     return user_dicts
 
 
+def get_previous_releases(project, start_version, limit=5):
+    # given a release version + project, return the previous
+    # `limit` releases (includes the release specified by `version`)
+    try:
+        release_dates = Release.objects.filter(
+            organization_id=project.organization_id,
+            version=start_version,
+            projects=project,
+        ).values('date_released', 'date_added').get()
+    except Release.DoesNotExist:
+        return []
+
+    start_date = release_dates['date_released'] or release_dates['date_added']
+
+    return list(Release.objects.filter(
+        projects=project,
+        organization_id=project.organization_id,
+    ).extra(
+        select={
+            'date': 'COALESCE(date_released, date_added)',
```

Comment:
could possibly alias this value and reference the alias in the where clause

Reply:
although this is not really part of the changeset

Reply:
@alex-hofsteede i think sql and/or django doesnt play nice with that- i forget which